sway/commands/show_marks.c

```diff
@@ -2,6 +2,8 @@
 #include <string.h>
 #include "sway/commands.h"
 #include "sway/config.h"
+#include "sway/tree/root.h"
+#include "sway/tree/root.h"
```

Comment:
`sway/tree/root.h` only needs to be added once

Reply:
Derp. Fixed by replacing the commit.